### DISCUSSION TITLE
Welcome to nostr-java Discussions!

### DISCUSSION THREAD

**Discussion by tcheeric** [tcheeric/nostr-java#64] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MHammad33** [comment]
Hi 👋 I am M Hammad Afzal from Pakistan pursuing a software engineering degree from NUST. I would like to know about Java. Why it is worth it and why should we learn it. Also, discussion is a great platform to connect with the community. Looking forward to know more about discussions and connecting with the community.

**Comment by cyborr** [comment]
Hi, I would like to know the right way to use nostr-java on Android Studio. Thanks

**Reply by kuiperanon** [reply]
Hi, I am trying to use nostr-java with Android. I believe there are some updates to nostr-java required before it will be able to work with android apps. 

Here is one issue that must be solved before it can work on android: https://github.com/tcheeric/nostr-java/issues/120

**Comment by dukeh3** [comment]
Do you guys run any telegram channel or so?

**Comment by tcheeric** [comment]
Hi there. No, we don't.
-- 
 Secured with Tuta Mail: 
 https://tuta.com/free-email


2 Jun 2025, 16:51 by ***@***.***:
